src/omni/embed.py

```diff
@@ -273,17 +288,36 @@ def get_filter_search_param_info(
         if not isinstance(values, list):
             values = [values]
         filter_key = f"f--{self.field}"
-        filter_value = [
-            json.dumps(
-                {
-                    "is_inclusive": False,  # FUTURE: Set this dynamically if needed for future operators.
-                    "is_negative": False,
-                    "kind": self.operator.value,
-                    "type": self.type.value,
-                    "values": values,
-                }
-            )
-        ]
+
+        is_inclusive = False
+
+        operator_kind = self.operator.value
+
+        if self.operator == self.Operator.greater_than_or_equal:
+            is_inclusive = True
+            operator_kind = self.Operator.greater_than.value
+        elif self.operator == self.Operator.less_than_or_equal:
+            is_inclusive = True
+            operator_kind = self.Operator.less_than.value
+
+        filter_value_param = (
```

Comment:
Having the base values for this duplicated is kind of hard to read and we have to keep them in sync as we update.

```suggestion
        filter_value_param = {
                "is_negative": self.is_negative,
                "kind": operator_kind,
                "type": self.type.value,
                "values": values,
         }
         if self.type == self.Type.number:
             filter_value_param['is_inclusive'] = is_inclusive
```

Reply:
This is a good suggestion, I'll update.